### DISCUSSION TITLE
Please help me to collect interesting video games!

### DISCUSSION THREAD

**Discussion by yokoro13** [NAIST-SE/AttractingContributors#85] [General]
Please tell me your favorite video games!

https://github.com/yokoro13/AttractingContributors-VideoGames/blob/main/README.md

**Comment by teruto725** [comment]
Added Minecraft to your repository!
Check Please!

**Reply by yokoro13** [reply]
Thank you for your contribution!

**Comment by sergiodl-naist** [comment]
Added on more!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers

**Comment by iikyara** [comment]
Added Getting Over It with Bennett Foddy!
Please help me too!
https://github.com/iikyara/AttractingContributors-Gifts
